frontend/selectors´/index.js

```diff
@@ -0,0 +1,33 @@
+import { createSelector } from 'reselect';
```

Comment:
the selectors folder contained a trailing backtick. please changed this.

Reply:
fixed